apps/test/unit/signUpFlow/FinishStudentAccountTest.tsx

```diff
@@ -190,143 +180,139 @@ describe('FinishStudentAccount', () => {
+    expect(finishSignUpButton.getAttribute('aria-disabled')).toBe('true');
 
     // Enter state
     fireEvent.change(stateInput, {target: {value: 'WA'}});
 
-    // Error message doesn't show when state is selected
+    // Error does not show and button is enabled when state is entered
     expect(screen.queryByText(locale.state_error_message())).toBe(null);
+    expect(finishSignUpButton.getAttribute('aria-disabled')).toBe(null);
 
     // Clear state
     fireEvent.change(stateInput, {target: {value: ''}});
 
-    // Error shows with empty state
+    // Error shows and button is disabled with empty state
     screen.getByText(locale.state_error_message());
+    expect(finishSignUpButton.getAttribute('aria-disabled')).toBe('true');
   });
 
   it('parentEmail error shows if parent checkbox is selected and parentEmail is selected then cleared', () => {
     renderDefault();
+    const finishSignUpButton = screen.getByRole('button', {
+      name: locale.go_to_my_account(),
+    });
+
+    // Set other required fields
+    const displayNameInput = screen.getAllByDisplayValue('')[1];
+    const stateInput = screen.getAllByRole('combobox')[1];
+    const ageInput = screen.getAllByRole('combobox')[0];
+    fireEvent.change(displayNameInput, {target: {value: 'FirstName'}});
+    fireEvent.change(stateInput, {target: {value: 'WA'}});
+    fireEvent.change(ageInput, {target: {value: '6'}});
+
+    // Button is enabled after required fields are filled before parent checkbox is checked
+    expect(finishSignUpButton.getAttribute('aria-disabled')).toBe(null);
 
     // Check parent checkbox
     fireEvent.click(screen.getAllByRole('checkbox')[0]);
 
-    // Error message doesn't show by default
+    // Error message doesn't show and button is disabled after parent checkbox is checked
     const parentEmailInput = screen.getAllByDisplayValue('')[1];
     expect(screen.queryByText(locale.email_error_message())).toBe(null);
+    expect(finishSignUpButton.getAttribute('aria-disabled')).toBe('true');
 
     // Enter state
     fireEvent.change(parentEmailInput, {target: {value: 'parent@email.com'}});
 
-    // Error message doesn't show when state is selected
+    // Error does not show and button is enabled when email is entered
     expect(screen.queryByText(locale.email_error_message())).toBe(null);
+    expect(finishSignUpButton.getAttribute('aria-disabled')).toBe(null);
 
     // Clear state
     fireEvent.change(parentEmailInput, {target: {value: ''}});
 
-    // Error shows with empty state
-    screen.getByText(locale.email_error_message());
-  });
-
-  it('finish student signup button starts disabled', () => {
-    renderDefault();
-
-    const finishSignUpButton = screen.getByRole('button', {
-      name: locale.go_to_my_account(),
-    });
-    expect(finishSignUpButton.getAttribute('aria-disabled')).toBe('true');
-  });
-
-  it('finish student signup button is disabled until required fields are entered without errors', () => {
```

Comment:
I removed this general test that does not specifically test one particular field, instead these checks were split up across the tests above